roles/ansible_service_broker/tasks/install.yml

```diff
@@ -210,7 +210,7 @@
     kind: ConfigMap
     content:
       path: /tmp/cmout
-      data: "{{ lookup('template', 'configmap.yaml.j2') | from_yaml }}"
+      data: "{{ ansible_service_broker_full_broker_config_map }}"
```

Comment:
You could just default it here as well, like
```yaml
data: "{{ ansible_service_broker_full_broker_config_map | default(lookup('template', 'configmap.yaml.j2') | from_yaml) }}"

Reply:
Yes, it looks better like this. I have done it as you propose. Could you review again? Thanks!